tests/yaml/issue-615.yaml

```diff
@@ -1,4 +1,4 @@
-# see https://github.com/liblouis/liblouis/issues/615
+# Test for issue #615: The always opcode unworking after the numeral (https://github.com/liblouis/liblouis/issues/615)
```

Comment:
"unworking" is this a typo?

Reply:
I just took the title of the Github issue.

Reply:
I agree a better description would be better, but I didn't bother.

Reply:
OK